src/payment-flow-handlers/handlers/paypal-flow-handler.ts

```diff
@@ -84,28 +83,18 @@ export class PayPalFlowHandler
   }
 
   async payPalPaymentStarted(
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
     dataSource: PayPalButtonDataSourceInterface,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
     options: object,
   ): Promise<void> {
-    console.debug(
```

Comment:
paypal "feels like a black box." - these debugs can be extremely helpful. but at least the abstraction is there so that there is a place to isolate code when debugging.
(not being wishy washy, I am just noting value :hehe:)

Reply:
Yeah agreed. I'd kinda like to circle back once we've decided on a logging solution and re-add some of these. I left all of the `console.errors` to help us debug if anything pops up.